services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:

```

Comment:
-> `MovieSession`

Reply:
I was sure that both all() and filter() return QuerySet or am I wrong?

Reply:
you are right